it/TransmissionRequestWorkItemRepositoryISpec.scala

```diff
@@ -40,35 +42,38 @@ class TransmissionRequestWorkItemRepositoryISpec
     )
     .build()
 
+  override implicit val patienceConfig: PatienceConfig = PatienceConfig(
```

Comment:
adding the trait `IntegrationPatience` is usually enough

Reply:
Some of these tests had a 5 second timeout and others 10 so thought to preserve that behaviour